web/src/hash_util.ts

```diff
@@ -192,6 +186,15 @@ export function parse_narrow(hash: string): NarrowTerm[] | undefined {
     return terms;
 }
 
+export function channels_settings_edit_url(
```

Comment:
Moved this so that it would be with the other two functions for these streams/channels settings overlay hash helper functions, instead of up with the helper functions for `#narrows`.